front-dispatcher/src/App.tsx

```diff
@@ -1,15 +1,25 @@
+import { useAuth0 } from "@auth0/auth0-react";
 import React from "react";
 import { Route, Routes } from "react-router-dom";
 import LoginPage from "./features/LoginPage/LoginPage";
 
 function App() {
+  const { user, isAuthenticated, isLoading } = useAuth0();
+  console.log(user, isAuthenticated, isLoading);
+
   return (
     <div className="App">
-      <Routes>
-        <Route path="/login" element={<LoginPage />} />
-        <Route path="/dashboard" element={<p>dash</p>} />
-        <Route path="/" element={<LoginPage />} />
-      </Routes>
+      {!isAuthenticated ? (
+        <Routes>
```

Comment:
Its not mandatory but you can export this logic to components in the same file 

const RouteA = ()=>{.....}
const RouteB = ()=>{....}

{isAuthenticated? RouteA:RouteB}